config/apply/ccsm/translate.js

```diff
@@ -344,6 +344,20 @@ const procedureMappings = {
   'ECT.14001.8': 'Endocervical Curettage'
 };
 
+// EPIC Code System for EpisodeOfCare Type
+const episodeOfCareTypeCodeSystem = [
+  'urn:oid:1.2.840.114350.1.13.88.2.7.2.726668', // PROD
+  'urn:oid:1.2.840.114350.1.13.88.3.7.2.726668' // COP
+];
+
+const snomedCtCodeSystem = 'http://snomed.info/sct'
```

Comment:
Two proposed changes here:
1. The second code works for both BLD (the test environment where we ran all of mock patient tests) and COP, so it may be helpful to document both in the comment here.
2. Add semi-colon.

```suggestion
// EPIC Code System for EpisodeOfCare Type
const episodeOfCareTypeCodeSystem = [
  'urn:oid:1.2.840.114350.1.13.88.2.7.2.726668', // PROD
  'urn:oid:1.2.840.114350.1.13.88.3.7.2.726668' // BLD & COP
];

const snomedCtCodeSystem = 'http://snomed.info/sct';
```